CharacterCreation/02_Classes.txt

```diff
@@ -512,15 +710,28 @@ Class Feats:
 ======================================
 
 =====   BESERKER   =====
-As an aficionado of the combat arts, you understand how important tactile feedback 
-is. Guns simply don't offer the appropriate amount, which is why you prefer to 
-BEAT YOUR ENEMIES TO DEATH USING YOUR BARE HANDS WHENEVER POSSIBLE!!!!!!!!!!!!!!!!!!!! MORE EXCLAMATION MARKS!!!!!!!!!!!!!!!! 
+	As an aficionado of the combat arts, you understand how important tactile 
+feedback is. Guns simply don't offer the appropriate amount, which is why you 
+prefer to BEAT YOUR ENEMIES TO DEATH USING YOUR BARE HANDS WHENEVER 
+POSSIBLE!!!!!!!!!!!!!!!!!!!! MORE EXCLAMATION MARKS!!!!!!!!!!!!!!!! 
+
 Class Primary Attribute: Strength
-Passives: Expert Unarmed
-Weapon Proficiencies: Pistols.
-Armor Proficiencies: Concealed, Recon, Light, Medium, Medium-Heavy.
 Starting Wealth: $2170 + 100*Luck
-Class Feats:
+
+Passives:
+* Gain Expert in Unarmed
```

Comment:
Can you change this to something like , "Gain usage of melee weapon "Hand-to-Hand [Expert]", disregarding requirement"?

I don't think there is an "Expert in Unarmed" and I believe "Unarmed Expert" was changed.

(If this isn't too much scope creep)

Reply:
wilco